.github/PULL_REQUEST_TEMPLATE/version_release.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Version Release
+about: Please use this pull request template when preparing for a version release.
+---
+
+## Release Version
+<!--Please indicate what version you are preparing for release below.-->
+
+
+## Further Steps
+After this pull request has been approved and merged, certain steps should be completed. 
+These are listed in order below for your convenience:
+- Create release for this version via GitHub.
+  - Use this version for release tag and title (i.e, v1.3.0).
+  - Write short description of changes and copy CHANGELOG for this version for body.
```

Comment:
We should also make a note so that when doing a release that contains breaking changes to outline them and the possible solutions (if not automatically implemented) in the release notes.

Reply:
Updated!